src/content/docs/en/guides/cms/kontent-ai.mdx

```diff
@@ -0,0 +1,472 @@
+
+export const deliveryClient = createDeliveryClient({
+    environmentId: import.meta.env.KONTENT_ENVIRONMENT_ID,
+    previewApiKey: import.meta.env.KONTENT_PREVIEW_API_KEY,
+});
+```
+
+This implementation creates a new `DeliveryClient` using credentials from the `.env` file.
+
+*Note: The `previewApiKey` is optional. When used, you can [configure each query](https://github.com/kontent-ai/delivery-sdk-js#enable-preview-mode-per-query) to the Delivery API endpoint to return the latest versions of content items regardless of their state in workflow. Otherwise, only published items are returned.*
+
+Finally, your root directory should now include these new files:
+
+<FileTree title="Project Structure">
+- src/
+  - lib/
+    - **kontent.ts**
+- **.env**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+### Fetching data
+
+The `DeliveryClient` is now available to all components. To fetch content, use the `DeliveryClient` and fluent API syntax to define your desired items. This example shows a basic fetch of blog posts and renders their titles in a list:
+
+```astro
+---
+import { deliveryClient } from "../lib/kontent";
+
+const blogPosts = await deliveryClient
+    .items()
+    .type("blogPost")
+    .toPromise()
+---
+<ul>
+  {blogPosts.data.items.map(blogPost => (
+    <li>{blogPost.elements.title.value}</li>
+  ))}
+</ul>
+```
+
+You can find more querying options in the [Kontent.ai documentation](https://kontent.ai/learn/develop/hello-world/get-content/javascript).
+
+## Making a blog with Astro and Kontent.ai
+
+With the setup above, you are now able to create a blog that uses Kontent.ai as the source of content.
+
+### Prerequisites
+
+1. **Kontent.ai project** - For this tutorial, we recommend using a blank project. If you already have some content types in your content model, you may use them, but you will need to modify the code snippets to match your content model.
+2. **Astro project configured for content fetching from Kontent.ai** - see above for more details on how to set up an Astro project with Kontent.ai
+
+### Setting up content model
+
+In Kontent.ai, navigate to **Content model** and create a new content type with the following fields and values:
+
+* **Name:** Blog Post
+* Elements:
+	* Text field
+		* **Name:** Title
+		* **Element Required:** yes
+	* Rich text field
+		* **Name:** Teaser
+		* **Element Required:** yes
+		* **Allowed in this element:** only check Text
+	* Rich text field
+		* **Name:** Content
+		* **Element Required:** yes
+	* Date & time field
+		* **Name:** Date
+	* URL slug field
+		* **Name:** URL slug
+		* **Element Required:** yes
+		* **Auto-generate from:** select "Title"
+
+Then, click on **Save Changes**.
+
+### Creating content
+
+Now, navigate to **Content & assets** tab and create a new content item of type **Blog Post**. Fill the fields using these values:
+
+* **Content item name:** Astro is amazing
+* **Title:** Astro is amazing
```

Comment:
I think this might be confusing for newcomers to Kontent AI. Would it be a good idea to replace with following?

```suggestion
* **Content item name:** Astro
* **Title:** Astro is amazing
```